src/login/login.component.tsx

```diff
@@ -25,6 +28,12 @@ export default function Login(props: LoginProps) {
       "https://implementation-assets.sfo2.digitaloceanspaces.com/openmrs_logo_white_large.png"
   };
 
+  React.useEffect(() => {
+    if (document.activeElement !== usernameInputRef.current) {
+      passwordInputRef.current.focus();
```

Comment:
After they click on the visibility icon button, we focus the password input so they can keep typing